golang/trace/zipin_tracereporter_test.go

```diff
@@ -267,3 +269,63 @@ func BenchmarkBuildThrift(b *testing.B) {
 		buildZipkinSpan(span, annotations, nil, endpoint)
 	}
 }
+
+func TestBuildBinaryAnnotation(t *testing.T) {
+	s := "testString"
+	ii64 := int64(5)
+	ii32 := int32(5)
+	ii16 := int16(5)
+	i := 5
+	b := false
+	f32 := float32(5.0)
+	f64 := float64(5.0)
+	bs := []byte{4, 3, 2}
+
+	tests := []struct {
+		banns    tchannel.BinaryAnnotation
+		expected *gen.BinaryAnnotation
+	}{
+		{
+			tchannel.BinaryAnnotation{Key: "string", Value: s},
```

Comment:
usually you don't skip the struct field name in the anonymous struct, so it should look something like:

```go
tests := []struct {
  annotation tchannel.BinaryAnnotation
  expected *gen.BinaryAnnotation
}{
  {
    annotation: tchannel.BinaryAnnotation{...},
    expected: &gen.BinaryAnnotation{...},
  },
[...]
}
```